python/catkin/test_results.py

```diff
@@ -134,6 +135,12 @@ def test_results(test_results_dir):
                 print('Skipping "%s": %s' % (name, str(e)))
                 continue
             results[name] = (num_tests, num_errors, num_failures)
+            if show_verbose and (num_errors + num_failures > 0):
+                print("Test %s was failed" % (name))
```

Comment:
"Test %s was failed" does not sound good. Also the test name should be surrounded by single quotes. Suggestion: "Full test results for test '%s'".